server/sonar-web/src/main/js/apps/projectActivity/utils.js

```diff
@@ -101,7 +101,7 @@ export const generateCoveredLinesMetric = (
 export function generateSeries(
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor this function to reduce its Cognitive Complexity from 17 to the 15 allowed. ([javascript:S3776](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3776))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fgreg%2Foverview_to_ts&issues=AWEFHxVGEFfjxkQcQG6S&open=AWEFHxVGEFfjxkQcQG6S)

[ISSUE_KEY]: AWEFHxVGEFfjxkQcQG6S